pyproject.toml

```diff
@@ -9,18 +9,18 @@ homepage = "https://github.com/honeycombio/libhoney-py"
 repository = "https://github.com/honeycombio/libhoney-py"
 
 [tool.poetry.dependencies]
-python = ">=3.5, <4"
+python = ">=3.7, <4"
 requests = "^2.24.0"
 statsd = "^3.3.0"
 
 [tool.poetry.dev-dependencies]
-coverage = [{version = "^5", python = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <4"}]
-pylint = [{version = "^2", python = ">=3.5"}]
+coverage = "^6.4.4"
+pylint = [{version = "^2.13", python = ">=3.7,<4"}]
```

Comment:
I think we want to take out the py2 thing here.

Reply:
Poetry caught this for us! the latest pylint requires Python >=3.7.2, so we have to ask for the latest pylint that allows python 3.7.0